webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -97,7 +97,7 @@ const ActionsBar = ({
       ouiaId="console-dropdown-item"
       onClick={() => visit(foremanUrl(`/hosts/${hostFriendlyId}/console`))}
       key="console"
-      isDisabled={!isHostActive}
+      isDisabled={!computeId || !isHostActive}
```

Comment:
note from @marisvirik, can we add a tooltip to explain why is it disabled so the user will know what's happening and how to resolve it?

Reply:
If the console button is disabled, we should add a tooltip.
@Ron-Lavi I recall there were 2 reasons for the button being disabled - could you post them here?

e.g. "Disabled because ..."
Tagging doc team to come up with a proper microcopy.

Reply:
@MariSvirik @Ron-Lavi Maybe we can add something like `Resource does not support the console function`, no need to say `Disabled because...` as it is already grayed out.

Reply:
@AkshayGadhaveRH Great input. There are two cases in which the button is grayed out. 

1. The host is turned off. I suggest that in that case, the tooltip would say this: Console is unavailable due to the host being powered off
2. The compute resource doesn't support the console function. In this case, I'd like to implement your suggestion with a little tweak: Compute resource does not support the console function

Is the wording okay?

Reply:
@Dyrkon how about `Console is unavailable as the host is powered off`, for the first case?

Reply:
@AkshayGadhaveRH Sounds great. I'll get on it.